.github/pull_request_template.md

```diff
@@ -0,0 +1,49 @@
+<!---
+
+    Please initially open your PR as a draft until CI has passed and the
+    PR is ready for review. 
```

Comment:
I am not a fan of this policy. GitHub shows whether CI has passed. I don't think we need a manual indicator of this.

What happens in practice is that people open PRs, CI passes, and then they forget to de-draft.